pkg/net/libp2p/key.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"github.com/btcsuite/btcd/btcec"
+	secp "github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
I think `secp` is not the best alias for this package. It has a complicated structure but I think we can alias it as `decredsecp256k1` so we have the concatenation of the "implementation provider" and "curve type".

But, we don't need to import that package at all. We can just use:
```
"github.com/btcsuite/btcd/btcec/v2"
```